pages/cloud/using.md

```diff
@@ -1,5 +1,289 @@
+
+For more information about instances, flavors and images please see the official Openstack Guide:  
+> -  [OpenStack Documentation](https://docs.openstack.org/nova/latest/user/launch-instances.html)
+
+
+#### Flavors
+
+> ###### Standard
+
+> | Instance Type | vCPU | Memory | Storage |
+> | :---     |    ---:  |   ---: |    ---:|
+> | c1.m4    | 1 cores  | 4 GB   | 20 GB  |
+> | c2.m8    | 2 cores  | 8 GB   | 20 GB  |
+> | c4.m16   | 4 cores  | 16 GB  | 20 GB  |
+> | c6.m24   | 6 cores  | 24 GB  | 20 GB  |
+> | c8.m32   | 8 cores  | 32 GB  | 40 GB  |
+> | c16.m64  | 16 cores | 64 GB  | 60 GB  |
+> | c32.m128 | 32 cores | 128 GB | 120 GB |
+
+
+> ###### Advanced 
+
+!!! Note
+    Advanced or GPU Subscription required to access GPU or Fast-IO Flavors
+
+>> **GPU** 
+
+> | Instance Type | GPU | vCPU | Memory | Storage |
+> | :---     |  :----: |   ---:  |   ---: |    ---:|
+> | c8.m32.gpu16   | V100D-16Q | 8 cores  | 32 GB   | 60 GB  |
+> | c16.m64.gpu32  | V100D-32Q | 16 cores | 64 GB   | 120 GB |
+
+>> **FastIO**
+
+!!! Note
+    FastIO/NVME Storage is volatile (instance storage) and will be destroyed when the instance is deleted.
+
+> | Instance Type | vCPU | Memory | NVME Storage | Block Storage |
+> | :---     |    ---: |   ---:  |   ---: |     ---:|
+> | c4.m16.e100   | 8 cores   | 16 GB   | 100 GB | 20 GB  | 
+> | c8.m32.e250   | 16 cores  | 32 GB   | 250 GB | 40 GB  |
+> | c16.m64.e500  | 32 cores  | 32 GB   | 500 GB | 60 GB  |
+
+
+#### Images
+
+An **Image** is a stock copy of an operating system that you will provision your instance with.
+CCR provides a current set of the major Linux distributions which are updated perodically. 
+These are all stock images pulled directly from the various vendor repos. Due to license restrictions, CCR does not provide Microsoft Windows images.
+
+> |Operating System | Version | Build Date |
+> | :----: | ---: | ---: |
+> | CentOS | 7.9  | 2021-11-16|
+> | CentOS | 8-stream | 2022-09-13|
+> | CentOS | 9-stream | 2022-09-19|
+> | Debian | 10.13.4 | 2022-10-13|
+> | Fedora | 36 | 2022-05-04|
+> | Rocky  | 8.6 | 2022-07-02|
+> | Rocky  | 9.0 | 2022-08-30|
+> | Ubuntu  | 18.04 | 2022-10-14|
+> | Ubuntu  | 20.04 | 2022-10-18|
+> | Ubuntu  | 22.04 | 2022-10-18|
+
+### Creating an Instance in LakeEffect
+
+!!! Note
+    Before you can launch and login to instances, you must have setup a key pair as described [here](#keypairs)
+
+
+From the LakeEffect Dashboard go to **Compute -> Instances**
+
+Click on `Launch Instance` button at the top right corner. This will bring up the instance launcher which will guide you in creating your new instance by selecting your instance source, Flavor, Network, Security groups, and Keypair.
+
+!!! Note
+    _Your project has a quota set for the total number of instances you're permitted to run.  The chart on the right shows the current usage.  If you've hit your limit, please contact ccr-help@buffalo.edu to request assistance._
```

Comment:
we're adding a top level page called help.md so anywhere we say 'contact CCR Help' we can link to that

Reply:
replaced all the ccr-help@buffalo.edu with a link to the help doc.